code/game/gamemodes/changeling/powers/shuriken.dm

```diff
@@ -0,0 +1,73 @@
+/datum/action/changeling/shuriken
+	name = "Bone Shuriken"
+	desc = "We evolve the ability to break off shards of our bone, and shape them into 2 shurikens. Costs 15 chemicals per shuriken."
+	helptext = "This will damage our arms, and be noticable to nearbye people. Shurikens will be dulled after hitting someone. 30 chemicals are required to use this ability, but you'll only be charged 15 per shuriken you make, and refunded 15 for those you do not."
+	button_icon_state = "shuriken"
+	chemical_cost = 30 // covered in making the stars
+	dna_cost = 2
+	req_human = 1
+	max_genetic_damage = 20
+
+/obj/item/throwing_star/bone
+	name = "bone shuriken"
+	desc = "A gross shard of bone and flesh, sharpened to deadly proportions."
+	icon_state = "bone_star"
+	throwforce = 15
+	embedded_fall_chance = 5
+	embedded_impact_pain_multiplier = 3
+	embedded_unsafe_removal_pain_multiplier = 6
+	embedded_pain_chance = 10
+	embedded_pain_multiplier = 3 //If you can make a lot of these, they will not be as good.
+
+/obj/item/throwing_star/bone/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	..()
+	if(isliving(hit_atom))
+		name = "dull bone shuriken"
+		desc = "A gross shard of bone and flesh, now much duller than it used to be."
+		throwforce = 7
+		embed_chance = FALSE //No using these over and over, for the crew, or the clings.
+
+/datum/action/changeling/shuriken/proc/arm_check(var/obj/item/organ/external/O)
+	if(O && (!(O.status & ORGAN_SPLINTED)) && (!(O.status & ORGAN_BROKEN)))
+		return TRUE
```

Comment:
```suggestion
	if(O && (!(O.status & ORGAN_SPLINTED)) && (!(O.status & ORGAN_BROKEN)))
		return TRUE
	return FALSE
```

You can also be fancy and `if(O && !(O.status & (ORGAN_SPLINTED | ORGAN_BROKEN)))` but you decide for yourself if it's more readable.

By the way you can make bone shurikens out of robotic arms, since you don't check for `ORGAN_ROBOT` :upside_down_face:

Reply:
Oooh, good point about robotic arms, will correct that, thank you